src/spikeinterface/core/base.py

```diff
@@ -470,7 +471,32 @@ def clone(self) -> "BaseExtractor":
         return clone
 
     def check_if_dumpable(self):
-        return _check_if_dumpable(self.to_dict())
+        # Find all nested _is_dumpable
+        dict_repr = self.to_dict()
```

Comment:
Lets add explicitly to_dict(recursive=false) to avoid errors later

Reply:
good point, done